### PR TITLE
Implemented Claude 3.5 Sonnet 

### PR DESCRIPTION
**Pull request by cqdev-co**
## Description

Just added support for the newly released Claude 3.5 Sonnet AI Model for the submodule, currently working on getting it to work server-side but this allows others to test it out :) 

I also changed some leftover code from 'Continue' to 'PearAI' and fixed some links.

### CODE REVIEW

**Comment by nang-dev** [comment]
Freaky ass commit FR

**Comment by nang-dev** [comment]
i think you may need to pull main

**Comment by cqdev-co** [comment]
@nang-dev pulled main, also removed any mention of Claude 3.5 Sonnet Free Trial since that doesn't work. If we want to add it as a free trial it is easy to do so but need to set up the free-trial first

File: gui/src/pages/modelconfig.tsx [hunk 1]
```diff
@@ -171,9 +171,37 @@ function ModelConfig() {
             className="mt-2"
             source={modelInfo?.longDescription || modelInfo?.description}
           />
+          <br />
 
-          {modelInfo?.provider === "pearai-server" ? (
+          {(modelInfo?.collectInputFor?.filter((d) => d.required).length || 0) >
+            0 && (
             <>
+<<<<<<< HEAD
```
Comment (Fryingpannn) [inline gui/src/pages/modelconfig.tsx:179]: Rip merge conflicts

**Comment by nang-dev** [comment]
another banger from conor

**Comment by nang-dev** [comment]
Assuming this was tested

**Pull request merged**